model/src/main/kotlin/DependencyGraph.kt

```diff
@@ -102,7 +102,7 @@ data class DependencyGraph(
      * A list with the edges of this dependency graph. By traversing the edges, the dependencies of packages can be
      * determined.
      */
-    val edges: List<DependencyGraphEdge>? = null
+    val edges: Set<DependencyGraphEdge>? = null
```

Comment:
The comment in line 102 should now also say "set" instead of "list". Also, the commit message should probably also mention that order is not important either (even if a set usually maintains insertion order).